### ISSUE TITLE
Add support for monitoring an Application Load Balancer

### ISSUE THREAD

**Issue by Unshure** [cdklabs/cdk-monitoring-constructs#328]
### Feature scope

Application Load Balancer

### Describe your suggested feature

`cdk-monitoring-constructs` currently supports monitoring an applicationLoadBalanced Fargate Service, but not an Application Load Balancer by itself. Additionally, `cdk-monitoring-constructs` supports monitoring a Network Load Balancer by itself. I am requesting you add support to monitor an Application Load Balancer by itself.

**Comment by vkkis93** [comment]
Yeah, would be nice to have it

**Comment by echeung-amzn** [comment]
Duplicate of #46

**Comment by r0b0ji** [comment]
I am considering implementing this. I am happy to take requests on what people think are useful widgets for the ALB segment. I looked at the automatic dashboard and all metrics related to ALB in Cloudwatch. My initial thinking is to have 6 widgets (2 row, 3 column).

Following are the metrics I found interesting, but this exceeds the available slots I am planning. So, seeking some community suggestions on what you find more useful to see in dashboard.
1. RequestCount: Sum
2. ActiveConnectionCount: Sum, NewConnectionCount: Sum, RejectedConnectionCount: Sum (all on same widget)
3. ConsumedLCU: Avg
4. HTTPCode_ELB_4XX_Count: Sum, HTTPCode_ELB_5XX_Count: Sum (left/right)
5. ProcessedBytes: Sum
6. RequestCountPerTarget: Avg
7. UnhealthyHostCount: Sum
8. RequestCount: Sum
